packages/contracts-bedrock/test/vendor/eas/SchemaRegistry.t.sol

```diff
@@ -0,0 +1,287 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { console } from "forge-std/console.sol";
+import { ISchemaRegistry, SchemaRecord } from "src/vendor/eas/ISchemaRegistry.sol";
+import { ISchemaResolver } from "src/vendor/eas/resolver/ISchemaResolver.sol";
+import { ISemver } from "src/universal/interfaces/ISemver.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+contract SchemaRegistryTest is CommonTest {
+    // State variables
+    ISchemaRegistry registry;
+
+    // Events
+    event Registered(
+        bytes32 indexed uid,
+        address indexed registerer,
+        SchemaRecord schema
+    );
+
+    function setUp() public override {
```

Comment:
You don't need to define `registry` or call `setUp`, the `setUp` function in `CommonTest` will be called and `schemaRegistry` will exist as a global variable from your change in `Setup.sol`